sample/kotlin/src/main/cpp/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+project("datadog-native-sample")
 cmake_minimum_required(VERSION 3.10.2)
```

Comment:
shouldn't we update version here as well?

Reply:
Yes...I actually updated that on my end for some reason it did not reach into the PR 🤔